themes/Admin/css/admin.css

```
@@ -293,11 +293,17 @@ label.form-label {
 }
 
 fieldset {
-    border-bottom: solid 1px var(--bf-light-grey);
-    padding-bottom: 2rem;
+    border-top: solid 1px var(--bf-light-grey);
+    padding-bottom: 1rem;
+    padding-top:0.5rem;
     margin-bottom: 1rem;
 }
 
+fieldset.first {
```

```suggestion
fieldset:firstChild {
```

If we do it this way we don't have to manually add a `.first` class to each element.

---

I guess this `fieldset:firstChild` would target the first element within the fieldset, which is not what we want, since we want to target the first fieldset and not the subsequent ones.

What could work, would be targeting the first fieldset in the sequence, but only provided all fieldset elements are siblings: 

```css
fieldset:not(:first-of-type) {
    border-top: solid 1px var(--bf-light-grey);
}
``` 

That would be nice on one hand (no need to add a class on the first element), but also be more rigid. Like, if someone makes meta template which, e.g., puts the meta fieldsets in a grid, it would not work (the rule would target each fieldset, since those will no longer be siblings). 

So, what do you think?

---

That's a good point about flexibility. 

That's fine, we can do it this way.